<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlgWindow</class>
 <widget class="QMainWindow" name="AlgWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1293</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Algorithm</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>1181</width>
      <height>51</height>
     </rect>
    </property>
    <widget class="QPushButton" name="minButton">
     <property name="geometry">
      <rect>
       <x>1090</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="closeButton">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="logo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="id">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_btn">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>10</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="id_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>10</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Algorithm Editor</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>211</width>
      <height>651</height>
     </rect>
    </property>
    <widget class="QPushButton" name="oll_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>OLL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pll_btn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PLL</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>971</width>
      <height>651</height>
     </rect>
    </property>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>931</width>
       <height>631</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="oll_page">
      <widget class="QTableWidget" name="oll_table">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>931</width>
         <height>631</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>OLL</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Algorithm</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="pll_page">
      <widget class="QTableWidget" name="pll_table">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>931</width>
         <height>631</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>PLL</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Algorithm</string>
        </property>
       </column>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
